src/modules/dashboard/components/ColonyDecisions/DecisionPreview.tsx

```diff
@@ -0,0 +1,170 @@
+  }>();
+  const { walletAddress, username } = useLoggedInUser();
+  const userProfile = useUser(walletAddress);
+
+  const { data, error } = useColonyFromNameQuery({
+    // We have to define an empty address here for type safety, will be replaced by the query
+    variables: { name: colonyName, address: '' },
+    pollInterval: 5000,
+  });
+
+  if (error) console.error(error);
+
+  if (data?.processedColony) {
+    const { processedColony: colony } = data;
+
+    const UserAvatar = HookedUserAvatar({ fetchUser: false });
+
+    return (
+      <div className={styles.main}>
+        <div className={styles.upperContainer}>
+          <p className={styles.tagWrapper}>
+            <Tag text="Preview" appearance={{ theme: 'light' }} />
+          </p>
+        </div>
+        <hr className={styles.dividerTop} />
+        <div className={styles.contentContainer}>
+          <div className={styles.leftContent}>
+            <span className={styles.userinfo}>
+              <UserAvatar
+                colony={colony}
+                size="s"
+                notSet={false}
+                user={userProfile}
+                address={walletAddress || ''}
+                showInfo
+                popperOptions={{
+                  showArrow: false,
+                  placement: 'left',
+                  modifiers: [
+                    {
+                      name: 'offset',
+                      options: {
+                        offset: [0, 10],
+                      },
+                    },
+                  ],
+                }}
+              />
+              <span className={styles.userName}>{`@${username}`}</span>
+            </span>
+            <div className={styles.title}>
+              <Heading
+                tagName="h3"
+                appearance={{
+                  size: 'medium',
+                  margin: 'small',
+                  theme: 'dark',
+                }}
+                text={MSG.title}
+              />
+            </div>
+            <FormattedMessage
+              {...MSG.description}
+              values={{
+                h4: (chunks) => (
+                  <Heading
+                    tagName="h4"
+                    appearance={{ size: 'medium', margin: 'small' }}
+                    text={chunks}
+                  />
+                ),
+              }}
+            />
+          </div>
+          <div className={styles.rightContent}>
+            <div className={styles.buttonContainer}>
+              <Button
+                appearance={{ theme: 'secondary', size: 'large' }}
+                onClick={() => handleEdit()}
+                text={{ id: 'button.edit' }}
+                data-test="decisionEditButton"
+              />
+              <Button
+                appearance={{ theme: 'primary', size: 'large' }}
+                onClick={() => handleSubmit()}
+                text={{ id: 'button.publish' }}
+                style={{ minWidth: styles.wideButton }}
```

Comment:
I see `style={{ minWidth: styles.wideButton }} ` used a lot in the code but since I only want one of the buttons on the page to have the minWidth set, how can I do this via css from a parent component? I did try setting a className from the parent but it seemed to override the styles from appearance.

Reply:
THe way I see it (and I of course can be wrong) is when we can target css from the css file without adding any extra html we should do it via css file. If we need to add extra html elements to do that, then it's ok to do it inline. The goal is to avoid inline css as much as possible.
You can do in the css file the followong:
```
.buttonContainer > button:last-child {
  min-width: xxx;
}
```